### DISCUSSION TITLE
Quantisation

### DISCUSSION THREAD

**Discussion by jakemorgangit** [FundamentalFrequency/.github#17] [Ideas]
Incredible project and will be attempting to build as soon as I can source the parts.  Are there any plans to introduce a quantisation feature? I'm not terribly  talented and have awful timing so rely heavily on quantisation in DAWs to sync up my free hand clips.

**Comment by stonepreston** [comment]
Thats definitely something I plan to work on, although currently my main priority is upgrading the DAW to use the just-released Tracktion Engine 2.0.
